addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3791,7 +3791,7 @@ const SnippetOptionWidget = Widget.extend({
         if (moveUpOrLeft || moveDownOrRight) {
             // The arrows are not displayed if the target is in a grid and if
             // not in mobile view.
-            const isMobileView = this.$target[0].ownerDocument.defaultView.frameElement.clientWidth < 768;
+            const isMobileView = this.$target[0].ownerDocument.defaultView.frameElement.clientWidth < 992;
```

Comment:
@sobo-odoo Could you do better and use the utils to do this? config.js (legacy) / ui_service.js (new)? :+1: Ideally there are ways to write something like "if screenSize below or equal to MD", without re-hardcoding numbers.